build.gradle.kts

```diff
@@ -12,6 +12,7 @@ val kotlinVersion = "1.7.10"
 val jacksonVersion = "2.13.3"
 val springBootVersion = "2.7.2"
 val junitVersion = "5.9.0"
+val springSecurityVersion = "5.7.3"
```

Comment:
Where is this used? Or just leftover from something?

Reply:
Yep, at some point I thought I need it.